src/js/player.js

```diff
@@ -19,7 +22,7 @@ import log, { createLogger } from './utils/log.js';
 import toTitleCase, { titleCaseEquals } from './utils/to-title-case.js';
 import { createTimeRange } from './utils/time-ranges.js';
 import { bufferedPercent } from './utils/buffer.js';
-import * as stylesheet from './utils/stylesheet.js';
+// import * as stylesheet from './utils/stylesheet.js';
```

Comment:
Is this supposed to be commented out? If it is, it might be worth removing it all together

Reply:
It was one of those things when we tried to remove, created a compiler error. So we just left it.